packages/server/graphql/types/NewMeeting.ts

```diff
@@ -24,6 +24,45 @@ import Team from './Team'
 import TeamMember from './TeamMember'
 import TeamPromptMeeting from './TeamPromptMeeting'
 
+const isMeetingLocked = async (
```

Comment:
+1 not important, but I think this would be better placed in a helper function file to keep `NewMeeting` as short as possible and only focused on `NewMeeting` fields